lib/bulbs-elements/store/store.js

```diff
@@ -1,15 +1,13 @@
 import cloneObject from 'clone';
 import iterateObject from 'bulbs-elements/util/iterate-object';
-const DEBUG = false; // placeholder for real implementation
+import invariant from 'invariant';
+import { reduce } from 'lodash';
```

Comment:
Yeah I'm not sure if the form I used will work but I know there's a way to do it with just the methods you need.